### PR TITLE
Feature/storehouse crud

### PR DESCRIPTION
**Pull request by miikoo**

### CODE REVIEW

File: Rbyte/Src/Api/Rbyte.Api.Models/Discount/ApiDiscount.cs [hunk 1]
```diff
@@ -3,6 +3,6 @@
     public class ApiDiscount
     {
         public int DiscountId { get; set; }
-        public int Value { get; set; }
+        public decimal Value { get; set; }
```
Comment (ToKoAFC) [inline Rbyte/Src/Api/Rbyte.Api.Models/Discount/ApiDiscount.cs]: to powinien byc int

File: Rbyte/Src/Core/Rbyte.Application/Discount/DiscountService.cs [hunk 2]
```diff
@@ -22,27 +24,43 @@ public DiscountService(RbyteContext context)
...
         {
-            throw new System.NotImplementedException();
+            var dbDiscount = _context.Discounts.First(x => x.DiscountId == discountId);
+            _context.Discounts.Remove(dbDiscount);
         }
 
         public ApiDiscount Get(int discountId)
         {
-            throw new System.NotImplementedException();
+            var dbDiscount = _context.Discounts.First(x => x.DiscountId == discountId);
```
Comment (ToKoAFC) [inline Rbyte/Src/Core/Rbyte.Application/Discount/DiscountService.cs]: var discount = _context.Discounts.Where(x => x.DiscountId == discountId).Select(x => new ApiDiscount{
DiscountId = x.DiscountId,
 Value = x.Value
}).First();

return discount;

File: Rbyte/Src/Core/Rbyte.Application/Store/StorehouseService.cs [hunk 3]
```diff
@@ -23,27 +25,45 @@ public StorehouseService(RbyteContext context)
...
-            throw new NotImplementedException();
+            var dbStore = _context.Stores.First(x => x.StoreId == storehouseId);
+            _context.Stores.Remove(dbStore);
+            _context.SaveChanges();
         }
 
         public ApiStorehouse Get(int storehouseId)
         {
-            throw new NotImplementedException();
+            var dbStore = _context.Stores.First(x => x.StoreId == storehouseId);
```
Comment (ToKoAFC) [inline Rbyte/Src/Core/Rbyte.Application/Store/StorehouseService.cs]: poprawic jak wyzej

File: Rbyte/Src/Api/Rbyte.Api.Models/Discount/ApiDiscount.cs (hunk 1 above)
Comment (miikoo) [reply Rbyte/Src/Api/Rbyte.Api.Models/Discount/ApiDiscount.cs]: w bazie jest decimal

**Pull request merged**